tests/event_stream_rpc_server_connection_test.c

```diff
@@ -138,19 +139,28 @@ static int s_fixture_setup(struct aws_allocator *allocator, void *ctx) {
         .type = AWS_SOCKET_STREAM,
     };
 
-    struct aws_event_stream_rpc_server_listener_options listener_options = {
-        .socket_options = &socket_options,
-        .host_name = "127.0.0.1",
-        .port = 30123,
-        .bootstrap = test_data->server_bootstrap,
-        .user_data = test_data,
-        .on_new_connection = s_fixture_on_new_server_connection,
-        .on_connection_shutdown = s_fixture_on_server_connection_shutdown,
-        .on_destroy_callback = s_on_listener_destroy,
-    };
-
-    test_data->listener = aws_event_stream_rpc_server_new_listener(allocator, &listener_options);
-    ASSERT_NOT_NULL(test_data->listener);
+    /* Find a random open port */
+    uint16_t test_port = 0;
+    while (!test_data->listener) {
```

Comment:
If not too terribly difficult, wouldn't it be better to bind to port 0 (gets you an available port) and get the port number that you know is used? I'm thinking that in some CI environments, it's possible the test environment may not see another open port or that searching with random numbers can take an unstable period of time.

Reply:
I could try it. I don't believe `aws_socket` can do that right now cross-platform, so I hacked this together in the name of speed

Reply:
yeah, you can pass in zero as a port, but there's currently no way to query what port you actually ended up with